### DISCUSSION TITLE
How to add new row into the table

### DISCUSSION THREAD

**Discussion by ariannargesi** [TanStack/table#2971] [General]
Hi everyone
I know that this is a very basic question. But I couldn't find the answer by now.
In my application, I will fetch some data as an array from my graphql server and directly render it using react-table.
now I want to add a new row to this table. How should I do that?

Thank you very much for your helps

**Comment by rahulmathews** [comment]
There is a `date` props in the `ReactTable`, All you have to do is push an empty object or object with default values into the variable which `data` uses and you will have extra row.

**Reply by sovetski** [reply]
can you give an example of code please?

**Reply by rahulmathews** [reply]
I am not sure of the context, Can you post the issue and the pseudo code here and describe the issue? I can help you with this.
